sentry-android-gradle-plugin/src/main/groovy/io/sentry/android/gradle/SentryPlugin.groovy

```diff
@@ -144,7 +145,12 @@ class SentryPlugin implements Plugin<Project> {
      * @return
      */
     static String getDebugMetaPropPath(Project project, ApplicationVariant variant) {
-        return "${variant.mergeAssets.outputDir}/sentry-debug-meta.properties"
+        try {
+            return "${variant.mergeAssetsProvider.outputDir}/sentry-debug-meta.properties"
```

Comment:
`mergeAssetsProvider` has type `TaskProvider<MergeSourceSetFolders>`. I beleive we need call `get()`on it to make it works.